app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ Required ] Good job working on this feature so far, however, the project requirements state that before a user is able to log in one should be able to confirm the email is used for registration. Currently, your App logs someone in before verifying whether the email belongs to that person or not. Please enable `:confirmable` to add this functionality.

## Example code
```ruby
 devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable,
 :confirmable
```
## Project requirements
![Screenshot from 2023-10-18 23-49-16](https://github.com/melkamu12/microverse_blogapp/assets/16392046/449a8048-f2b1-4810-95d8-6b1206f3bd96)

Reply:
thank you i consider it.